mergedSchema.js

```diff
@@ -0,0 +1,82 @@
+import { mergeSchemas, introspectSchema, makeRemoteExecutableSchema } from "graphql-tools"
+import { ApolloLink } from "apollo-link"
+import { createHttpLink } from "apollo-link-http"
+import { setContext } from "apollo-link-context"
+import fetch from "node-fetch"
+
+import config from "./config"
+import localSchema from "./schema"
+
+function createConvectionLink() {
+  const httpLink = createHttpLink({
+    fetch,
+    uri: `${process.env.CONVECTION_API_BASE}/graphql`,
```

Comment:
Seems like this `uri` might be the main piece of convection-unique code here. I wonder if there's an opportunity to wrap this boilerplate in a way that'd make it easy to do just point to the urls we're stitching.

Reply:
The other Convection specific code is using the `convectionTokenLoader`. I agree that this can probably be made more generic, but was punting on that until we add another service.